### ISSUE TITLE
⌨ Accordion header pattern violation

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#6412]
## ⌨ Accessibility Issue

Our experience accordion violates the accordion pattern in the ARIA Authoring Practices Guide that states that the trigger button should be the only child in the accordion header.

[https://www.w3.org/WAI/ARIA/apg/patterns/accordion/#wai-ariaroles,states,andproperties:](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/#wai-ariaroles,states,andproperties:)

> Each accordion header button is wrapped in an element with role heading that has a value set for aria-level that is appropriate for the information architecture of the page.
> -   The button element is the only element inside the heading element. That is, if there are other visually persistent elements, they are not included inside the heading element.

## 📸 Screenshot

![image.png](https://images.zenhubusercontent.com/6172cf6e898e11fd20507433/7e2a137f-2dcd-493f-9f13-70623e6e6b42)

## 🙋‍♀️ Proposed Solution

The only solution I can think of is to design some sort of "accordion container" component so that the edit link could be outside the accordion header but still strongly associated with it.

There may be a technical solution where the edit link appears to be in the header but in fact is outside it, it in the DOM.  That seems like it violates that spirit of the pattern, if not a technical violation.

Or perhaps we need to move the edit link back inside the content or move away from using the accordion for experiences?

## ✅ Acceptance Criteria

- [ ] The button element is the only element inside the heading element. That is, if there are other visually persistent elements, they are not included inside the heading element.

**Comment by Jerryescandon** [comment]
Im adding @substrae cause it feels like this will need to be addressed at the component level everywhere.

**Comment by substrae** [comment]
@Jerryescandon @petertgiles

This ticket can be closed, as experience accordions were reworked (#6735) so that the actual accordion element is nested inside of card instead.
